src/privileged/multipreffer/api.js

```diff
@@ -29,21 +29,9 @@ this.multipreffer = class extends ExtensionAPI {
 };
 
 this.FirefoxHooks = {
-  init() {
-    AddonManager.addAddonListener(this);
-  },
-
-  get abortedPref() {
-    return `extensions.multipreffer.${gExtension.id}.aborted`;
-  },
-
+  _oldDefaultValues: {},
```

Comment:
So I guess this is necessary because you can't reset default preferences? Can you add a short comment on why we can't reset them?

Reply:
> So I kind of liked the idea of having both modes as options for setting prefs, but I'm fine with this.

Yeah, but I thought about it and since this add-on is specifically going to be used for studies, I think we should only support default-branch prefs (to correctly emulate real-world post-experiment state) until such a time as we actually want to test what happens when we set a user value for some pref. I'd rather not have us footgun ourselves scientifically due to having tons of options.

I'm wondering now if I should leave the existing code as it is in a separate branch rather than replacing it. I can spin off a new branch before merging this PR.

As for your review comments: I actually addressed all of them already in the search volume study add-on! I need to backport those changes into this repo.

I can't wait to have this be a node module so that forks don't get fragmented.

Thank you for the review!